docs/conf.py

```diff
@@ -224,7 +227,12 @@
 #latex_preamble = ''
 
 # Documents to append as an appendix to all manuals.
-#latex_appendices = []
+latex_appendices = [
+    'support',
+    'resources',
+    'changes',
+    'authors',
+]
```

Comment:
just seen: you only removed them from book.rst, so this only affects output generated from that.